go/vt/tabletserver/sqlquery.go

```diff
@@ -59,7 +59,7 @@ var stateName = []string{
 
 var (
 	// RPCErrorOnlyInReply is the flag to control how errors will be sent over RPCs for all queryservice implementations.
-	RPCErrorOnlyInReply = flag.Bool("rpc-error-only-in-reply", false, "if true, supported RPC calls will only return errors as part of the RPC server response")
+	RPCErrorOnlyInReply = flag.Bool("rpc-error-only-in-reply", true, "if true, supported RPC calls will only return errors as part of the RPC server response")
```

Comment:
I was leaving it in because we had some internal things that depend on it, and I was generally trying to do whatever I could to not break imports. Unfortunately, it didn't work out there are non-Go VtGate clients, so I'll need to fix those before I can get rid of this. That's what was causing the Travis failures.